tag_manager/content_security_policy.rst

```diff
@@ -158,7 +155,7 @@ If your tracking domain is custom, then you need to define it with ``img-src`` a
 .. code-block:: javascript
 
 	img-src <your-sources> your-custom-cpp-domain.com;
-	script-src <your-sources> your-custom-cpp-domain.com;
+	script-src <your-sources> your-custom-cpp-domain.com/ppms.js;
```

Comment:
what's cpp by the way?

Reply:
Client Piwik pro? :d

Reply:
Possible, but I am wondering what is the point. If this is confusing us, there is a chance it will confuse our clients. I am going to remove this.

Reply:
let's change it to `your-custom-domain.com`

Reply:
Done.